src/Misc/default-rapipdf.html.pode

```diff
@@ -0,0 +1,150 @@
+<!DOCTYPE html>
+<html lang='en' style='background-color: #01b0c4; overflow-y: hidden'>
+
+<head>
+    <title>RapiPdf Web Page</title>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href="https://fonts.googleapis.com/css?family=Montserrat:300,400,700|Roboto:300,400,700" rel="stylesheet">
+
+    <style>
+        body {
+            margin: 0;
+            padding: 0;
+            font-family: 'Montserrat', sans-serif;
+        }
+
+        .container {
+            max-width: 900px;
+            margin: 0 auto;
+            padding: 20px;
+        }
+
+        table {
+            width: 100%;
+            border-collapse: collapse;
+        }
+
+        th,
+        td {
+            padding: 10px;
+            border-bottom: 1px solid #ddd;
+        }
+
+        th {
+            text-align: left;
+        }
+
+        /* Style for the separate container for the RapiPdf element */
+        .rapipdf-container {
+            text-align: center;
+            margin-top: 20px;
+            /* Add some space between the form and the RapiPdf element */
+        }
+    </style>
+</head>
+
+<body style='color: white; font-family: Helvetica,Arial,sans-serif;'>
+    <div class="container">
+        <h1>RapiPdf Configuration</h1>
+
+        <!-- Form to set RapiPdf properties -->
+        <form id="rapipdf-form">
+            <table>
+                <tr>
+                    <th>Description</th>
+                    <th>Value</th>
+                </tr>
+                <tr>
+                    <td>Color used for headings of main sections in PDF</td>
+                    <td><input type="color" id="pdf-primary-color" name="pdf-primary-color" value="#b44646"></td>
+                </tr>
+                <tr>
+                    <td>Color used for sub headings</td>
```

Comment:
Just so it matches with the rest of Pode, use "Colour" instead :)

I'd also use "sub-headings" with a "-" as well.